zwave_js_server/util/node.py

```diff
@@ -92,23 +95,35 @@ async def async_bulk_set_partial_config_parameters(
     """Bulk set partial configuration values on this node."""
     config_values = node.get_configuration_values()
     property_values = [
-        value for value in config_values.values() if value.property_ == property_
+        value
+        for value in config_values.values()
+        if value.property_ == property_ and value.property_key is not None
     ]
 
     # If we can't find any values with this property, the property is wrong
     if not property_values:
+        # If we find a value with this property_, we know this value isn't split
+        # into partial params
+        if get_value_id(node, CommandClass.CONFIGURATION, property_) in config_values:
+            if isinstance(new_value, dict):
+                raise ValueTypeError(
+                    f"Configuration parameter {property_} for node {node.node_id} "
+                    "does not have partials"
+                )
+            # If the new value is provided as an int, we may as well try to set it
+            # using the standard utility function
+            else:
+                _LOGGER.info(
+                    "Falling back to async_set_config_parameter because no partials "
+                    "were found"
+                )
+                return await async_set_config_parameter(node, new_value, property_)
```

Comment:
mypy is complaining about this line:
`zwave_js_server/util/node.py:120: error: Incompatible return value type (got "Tuple[ConfigurationValue, CommandStatus]", expected "CommandStatus")`

Reply:
good catch thanks